src/plasma/plasma_manager.cc

```diff
@@ -670,7 +682,10 @@ void process_data_chunk(event_loop *loop,
                         int events) {
   /* Read the object chunk. */
   ClientConnection *conn = (ClientConnection *) context;
-  PlasmaRequestBuffer *buf = conn->transfer_queue.front();
+  PlasmaRequestBuffer *buf = conn->data_transfer_queue.front(); 
+  assert(buf != NULL);
+  assert(buf->type == MessageType_PlasmaDataReply);
+
```

Comment:
Do we need these asserts? Looks like type is set to `MessageType_PlasmaDataReply` in `process_data_request`.